SafarikaiEngine/SettingsManager.swift

```diff
@@ -41,6 +43,16 @@ class SettingsManager {
         set(value) { sharedUserDefaults.set(value, forKey: Keys.translationShow) }
     }
 
+    var shouldLookupOnlyOnHotkey: Bool {
+        get { sharedUserDefaults.value(forKey: Keys.shouldLookupOnlyOnHotkey) as? Bool ?? true }
```

Comment:
Should we set this to `false` as default? People that are used to the old behavior might be suprised and thinking the lookup is not working anymore if the default behavior is to translate only when Option is pressed.

Reply:
Absolutely